samples/bluetooth/mesh/dfu/target/README.rst

```diff
@@ -29,6 +29,13 @@ For uploading an image to the Distributor, this sample also requires a smartphon
 * `nRF Device Manager mobile app for Android`_
 * `nRF Device Manager mobile app for iOS`_
 
+Point-to-point DFU requirements
+*******************************
+
+The configuration overlay :file:`overlay-ptp_dfu.conf` enables the :ref:`dfu_over_ble` feature.
+
+This feature can be used as a backup option for the DFU process if the mesh network is not available.
```

Comment:
Could we clarify usage of them a bit? I mean impossibility to use in parallel. Or for example that, one procedure will not pick up dfu process of another if it was interrupted\suspended.

Reply:
@mia-ko : I changed the text here, could you check what I wrote?